### PR TITLE
Initialising project structure

### PR DESCRIPTION
**Pull request by YMonnier**
- created three more modules
    - `presentation`
    - `domain`
    - `data`
- `app` module depends on `presentation`, `data` - `data` module depends on `domain` module

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion compose_version
+        kotlinCompilerVersion '1.4.10'
+    }
+}
+
+dependencies {
+    implementation project(':domain')
```
Comment (axlefranc) [inline app/build.gradle:43]: Don't you need data to setup DI?

File: app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    kotlinOptions {
+        jvmTarget = '1.8'
+        useIR = true
+    }
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion compose_version
+        kotlinCompilerVersion '1.4.10'
```
[the remark this answers is not in the document]
Comment (axlefranc) [reply app/build.gradle:38]: Kotlin 1.4.20 was just released, we can use it.

[the remark this answers is not in the document]
Comment (YMonnier) [reply app/build.gradle:38]: there is an issue: https://github.com/android/compose-samples/issues/290

**Review by alexandre-lefranc — APPROVED** [review]
Many files need an empty line at the end. Do not forget to set this setting in AS.

**Pull request merged**